docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -42,42 +42,44 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
 
 ### Parameters
 
+* **grouping_expression**
+
+    Specifies the criteria based on which the rows are grouped together. The grouping of rows is performed based on
+    result values of the grouping expressions. A grouping expression may be a column alias like `GROUP BY a`, a column position like
+    `GROUP BY 0`, or an expression like `GROUP BY a + b`.
+   
 * **GROUPING SETS**
 
     Groups the rows for each subset of the expressions specified in the grouping sets. For example,
     `GROUP BY GROUPING SETS (warehouse, product)` is semantically equivalent
-    to union of results of `GROUP BY warehouse` and `GROUP BY product`. This clause
+    to union of results of `GROUP BY warehouse` and `GROUP BY product`. 
+    `GROUP BY GROUPING SETS ((warehouse, product), (product), ())` is semantically equivalent to union
```

Comment:
We should move this to the end, because the next sentence `performs aggregation of subset of the columns specified in the GROUPING SETS clause.`

If we specify groups directly instead of column names, we do not get all the subsets anymore.

Reply:
Done